lib/portage/sync/modules/git/git.py

```diff
@@ -212,7 +212,19 @@ def verify_head(self, revision='-1'):
 				writemsg_level("!!! Verifying against specified key requires gemato-11.0+ installed\n",
 					level=logging.ERROR, noiselevel=-1)
 				return False
-			openpgp_env = gemato.openpgp.OpenPGPEnvironment()
+
+			proxy = None
+			# Set global proxy setting to Gemato ephemeral env
+			if 'http_proxy' in os.environ:
+				proxy = os.environ['http_proxy']
```

Comment:
@zmedico: should this be feed from `os.environ` or `spawn_kwargs`? I'm not sure to see if `spawn_kwargs` is initially feed by `os.environ`.